### ISSUE TITLE
Support polls / vote

### ISSUE THREAD

**Issue by luchmhor** [wee-slack/wee-slack#667]
I noticed that slack allows to create polls. I have not found out how to create a poll using wee-slack, neither how to vote on existing polls.
Is it doable?

**Comment by trygveaa** [comment]
You can create a poll by running:
```
/slack slash /poll "Some poll text" "Option 1" "Option 2"
```

I don't think you can vote in polls though. I looked through Slacks API documentation, but I didn't find any method to trigger message buttons. So if we want to implement it, we may have to check what the web client does, and see if we can do the same (I'm generally not a fan of doing things outside of the API though).

**Comment by tfheen** [comment]
https://api.slack.com/interactive-messages#responding seems relevant. Not sure how much we're into well-documented territory here, though.

**Comment by trygveaa** [comment]
Yeah, I saw that. I haven't read it in detail, but it only seems to describe how you create a message with buttons, and how you can respond when a user clicked a button you created. It doesn't say anything about triggering a button as far as I can see.

**Comment by aereaux** [comment]
To add to this: previously, the votes in the poll were shown, but recently it started only showing a message about there being a poll.  Is there anything that can be done about that?

**Comment by auscompgeek** [comment]
@aereaux you'll have to give more details; what poll app are you referring to?

Either way, I hate the idea of these poll apps. Reactions work just fine for polls, and is even suggested by Slack for that purpose.

**Comment by aereaux** [comment]
I'm fairly sure it's called Polly.  That's the username that is shown.  The change happened sometime on Friday.

I agree about these poll apps, but I don't think it would be easy to get the rest of my team to switch.

**Comment by trygveaa** [comment]
With #729 polls are now shown again.

Responding to polls is still not possible unfortunately, and I still don't know if it's possible. I checked the web client, and it uses an undocumented API endpoint which only works with the tokens the web client use, and not with oauth tokens, so we can't use that.
